tests/test_package.py

```diff
@@ -1071,3 +1071,35 @@ def test_package_validation_duplicate_resource_names_issue_942():
     errors = package.metadata_errors
     assert len(errors) == 1
     assert errors[0].note == "names of the resources are not unique"
+
+
+def test_package_set_hashing():
+    package = Package(hashing="SHA-1")
+    assert package.hashing == "SHA-1"
+
```

Comment:
Let's try not to use empty lines here too. It's not worse or better just let's be consistent with the codebase style